common/src/main/scala/explore/model/RootModel.scala

```diff
@@ -26,6 +26,7 @@ case class RootModel(
   tabs:                           EnumZipper[AppTab],
   localPreferences:               ExploreLocalPreferences,
   focusedObs:                     Option[FocusedObs] = none,
+  focusedTarget:                  Option[Target.Id] = none,
```

Comment:
I couldn't decide whether I should "unwrap" the `FocusedObs` or wrap the focused target to make them consistent. I know we had some discussion about `FocusedObs` when we adopted unshared targets and got rid of the focused ADT in a previous incarnation of the target model, but I don't remember why we decided to keep the wrapper. At this point, it seems like overkill to me. So, at the moment the focused observation id is wrapped in `FocusedObs` and the focused target id is used directly.

Reply:
`FocusedObs` may be vestigial from the old ADT, maybe we should remove it too and make things uniform?

Reply:
I agree. I'll remove it.